openslides/assignments/static/templates/assignments/assignment-detail.html

```diff
@@ -53,6 +53,13 @@ <h3 translate>Number of posts to be elected</h3>
           {{ assignment.open_posts }}<br>
       </div>
       <div class="col-md-6">
```

Comment:
@emanuelschuetze Please check template here. We have three times `col-md-6`. This should be `col-md-4`, shouldn't it?